docs/source/ptd-file-format.md

```diff
@@ -0,0 +1,144 @@
+# `.ptd` file format
+
+ExecuTorch `.ptd` files are serialized as modified binary flatbuffer
+files with data segments appended. They provide a way to store named data using
+the FlatTensor format. Named data can be tensors or opaque blob data (usually for backends that do not expose data format).
+
+Code related to the PTD file format is in the `//executorch/extension/flat_tensor/` directory.
+
+```
+             ┌───────────────────────────────────┐
+             │Standard flatbuffer header         │
+             ├───────────────────────────────────┤
+             │ExecuTorch extended header         │
```

Comment:
doesn't executorch's extended header come before any flatbuffer? From what i remember executorch's extended header pointed to the first offset of flatbuffer